examples/model_compress/fpgm_torch_mnist.py

```diff
@@ -16,7 +16,8 @@ def forward(self, x):
         x = F.max_pool2d(x, 2, 2)
         x = F.relu(self.conv2(x))
         x = F.max_pool2d(x, 2, 2)
-        x = x.view(-1, 4 * 4 * 50)
+        #x = x.view(-1, 4 * 4 * 50)
+        x = x.view(64, -1)
```

Comment:
typically change it to x.view(x.size(0), -1)

Reply:
good suggestion, thanks!